Zero_engine.alp

```diff
@@ -56554,6 +56000,93 @@ public class J_AVGC_data implements Serializable {
 		return super.toString();
 	}
 
+	/**
+	 * This number is here for model snapshot storing purpose<br>
+	 * It needs to be changed when this class gets changed
+	 */ 
+	private static final long serialVersionUID = 1L;
+
+}]]></Text>
+		</JavaClass>
+		<!--   =========   Java Class   ========  -->
+		<JavaClass>
+			<Id>1726148464063</Id>
+			<Name><![CDATA[J_Address]]></Name>
+			<Text><![CDATA[/**
+ * J_Address
+ */	
+public class J_Address implements Serializable {
+
+	private String streetName;
+	private int houseNumber;
+	private String houseLetter;
+	private String houseAddition;
+	private String postalcode;
+	private String city;
+	
+    /**
+     * Default constructor
+     */
+    public J_Address() {
+    }
+    
+    public String getAddress() {
+		String text = this.streetName + " " + this.houseNumber;
+		if ( this.houseLetter != null ) {
+			text = text + " " + this.houseLetter;
+		}
+		if ( this.houseAddition != null ) {
+			text = text + " " + this.houseAddition;
+		}
+		text = text + ", " + this.postalcode + " " + this.city;
+		return text;
+    }
```

Comment:
(hoeft niet gewijzigd, enkel een FYI)

Kan eventueel met StringBuilder of `text += " "`

```java
var sb = StringBuilder();

sb.append(this.streetName);
sb.append(" ")
sb.append(this.houseNumber);

if ( this.houseLetter != null ) {
    sb.append(" ");
    sb.append(this.houseNumber);
}

if ( this.houseAddition != null ) {
    sb.append(" ");
    sb.append(this.houseAddition);
}

sb.append(", ");
sb.append(this.postalcode);
sb.append(" ");
sb.append(this.city);

return sb.toString();
```